packages/appkit/tests/siwe.test.ts

```diff
@@ -21,6 +21,19 @@ describe('SIWE mapped to SIWX', () => {
   let appkit: AppKit
 
   beforeEach(async () => {
+    global.document = {
```

Comment:
Can we use  `mockWindowAndDocument` method which is mocking document and window object, if it's not doing the exact same, we could extend it maybe?